tests/test_inject_uuids.py

```diff
@@ -0,0 +1,245 @@
+from uuid import UUID
+from mbtools.models import MoodleQuestionBank
+from mbtools.inject_uuids import main
+
+
+def validate_uuid4(uuid_string):
+    try:
+        UUID(uuid_string, version=4)
+        return True
+
+    except ValueError:
+        return False
+
+
+def test_inject_uuids(tmp_path, mocker):
+    questionbank_xml = """
+<?xml version="1.0" encoding="UTF-8"?>
+<question_categories>
+  <question_category id="1">
+  <idnumber>$@NULL@$</idnumber>
+    <question_bank_entries>
+      <question_bank_entry id="1">
+        <idnumber>$@NULL@$</idnumber>
+        <question_version>
+          <question_versions>
+            <version>1</version>
+            <questions>
+              <question id="question1v1">
+                <qtype></qtype>
+              </question>
+            </questions>
+          </question_versions>
+          <question_versions>
+            <version>2</version>
+            <questions>
+              <question id="question1v2">
+                <qtype></qtype>
+              </question>
+            </questions>
+          </question_versions>
+        </question_version>
+     </question_bank_entry>
+      <idnumber>$@NULL@$</idnumber>
```

Comment:
This is a benign / minor issue, but in so much as it might confuse our future selves: I think this element is just hanging around for all three tests and can be removed without any impact since in practice there is no `<idnumber>` child element under `<question_bank_entries>`.

Reply:
Oh, no, I meant this `<idnumber>$@NULL@$</idnumber>` line between the entries.